librina/src/rib_v2.cc

```diff
@@ -1932,6 +1931,7 @@ void RIBDaemon::remote_open_connection_result(const cdap_rib::con_handle_t &con,
 	// FIXME remove invoke_id
 
 	app_con_callback_->connectResult(res, con);
+	store_connection(con);
```

Comment:
I don't understand why opening a connection (application Manager -> Mad) has to store the connection; it shouldn't, if I am not wrong. The store_connection is to bind port_id to RIB for incoming requests (RIB MAd) and in this case the manager doesn't necessarily has to expose a RIB (in a general case).

Why is that?